lib/load_balancer.dart

```diff
@@ -69,8 +69,8 @@ class LoadBalancer implements Runner {
   /// If [timeout] and [onTimeout] are provided, they are forwarded to
   /// the runner running the function, which will handle a timeout
   /// as normal.
-  Future<R> run<R, P>(R function(P argument), argument,
-      {Duration timeout, onTimeout(), int load: 100}) {
+  Future<R> run<R, P>(FutureOr<R> function(P argument), argument,
```

Comment:
Suggest consistently change P => T or don't change P => T.

Reply:
Change T's to P's.